new/release_and_deprecation_policy.md

```diff
@@ -1,20 +1,16 @@
 ---
 RFC: unassigned
-Title: Chef Major Version Release and Deprecation Policy
+Title: Chef Deprecation Policy
 Author: Tim Smith <tsmith@chef.io>
 Status: Draft
 Type: <Process>
 ---
 
-# Chef Major Version Release and Deprecation Policy
+# Chef Deprecation Policy
 
-A policy for shipping major Chef releases on a predictable schedule along with the communication of breaking changes in those releases to users.
+A policy for the the inclusion and communication of breaking changes.
 
-This RFC is two fold:
-
-It states that Chef will ship a new major release of the chef-client on a yearly schedule, while continuing to ship minor releases on the previous monthly schedule.
-
-It also sets policy for the for how and when breaking changes will be included and communicated to users. Chef will provide a clear road map of upcoming compatibility breaks, including remediation steps users can take to update their codebase. This will allow the project to progress while also providing users with the information necessary to scope future upgrades. It avoids what has been called the Chef 13 problem, in which the large time between major releases creates a mass of breaking changes that hinders adoption of new major releases.
+This RFC sets policy for how and when breaking changes will be included in Chef and how they will be communicated to users. Chef will provide a clear road map of upcoming compatibility breaks, including remediation steps users can take to update their codebase. This will allow the project to progress while also providing users with the information necessary to scope future upgrades. It avoids what has been called the Chef 13 problem, in which the large time between major releases creates a mass of breaking changes that hinders adoption of new major releases.
```

Comment:
Hmm. I think the real world example is useful here. People who aren't involved at the time can look at what happened in the chef 12 cycle and understand the problems.